### PR TITLE
Add a README

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Comment by anki-code** [comment]
Wow! You rock guys! I'm using [xonsh.AppImage](https://github.com/xonsh/xonsh/pull/3362) for this but your goals are greater! It's incredible!

Just for your info. When I'm using stand alone `xonsh.AppImage` I fix the paths and use pip without installation like a charm:
```
bash$ # run xonsh.AppImage on the host without python installed
bash$ ./xonsh.AppImage
xonsh> python
ImportError: No module named site
xonsh> $PATH = [$PYTHONHOME + '/bin'] + $PATH
xonsh> $PIP_TARGET='/tmp/xonsh/pip'
xonsh> import sys
xonsh> sys.path.append('/tmp/xonsh/pip')
xonsh> pip install pandas
xonsh> import pandas as pd
xonsh> # success
xonsh> python
xonsh> # success
```

It will be great if I can use minixonsh in https://github.com/xxh/xxh

**Comment by anki-code** [comment]
It will be great if I can use it in https://github.com/xonssh/xxh

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+* Very fast to start, compile and execute
+* Ability to run both scripts and have an interactive prompt
+* Runs natively on Linux, macOS, Windows (distribute as a single binary)
+* Calling programs
+* If statements
+* Environment variables (including `$PATH`)
+
+Down the road we would like to implement:
+
+* Larger subset of Python and Xonsh that can be statially compiled (such as
```
Comment (gforsyth) [inline README.md]:
```suggestion
* Larger subset of Python and Xonsh that can be statically compiled (such as
```

**Review by gforsyth — CHANGES_REQUESTED** [review]
small typo. otherwise looks good.

Also, what are you two up to?

**Comment by certik** [comment]
@gforsyth thanks for the review. @scopatz ok to merge now?

> Also, what are you two up to?

I would like to make xonsh fast. :)

**Pull request merged**